### DISCUSSION TITLE
接上了mirai-onebot但是不能接受和发送消息

### DISCUSSION THREAD

**Discussion by HanYaaaaaaa** [nonebot/discussions#428] [问答]
![HG9DNYNR5ST_A053%I0 38L](https://github.com/nonebot/discussions/assets/145742863/0ce25ad5-71f0-4fd3-9bf6-23ce462845ff)
![image](https://github.com/nonebot/discussions/assets/145742863/377e671c-6370-467c-9c88-eee8f1228e5f)

**Comment by HanYaaaaaaa** [comment]
mirai端是可以正常使用的

**Comment by yanyongyu** [comment]
日志内并没有显示出你连接成功

**Comment by A-n-k-a** [comment]
我也有这个问题，运行nonebot后mirai console显示`正向Websocket服务端 / 成功连接`，但是nonebot日志没有连接成功的字样，且无法收发消息，另测试发现mirai端设置有access_token，但nonebot端未设置access_token，运行nonebot后mirai console仍然会显示`正向Websocket服务端 / 成功连接`，同样nonebot日志没有连接成功的字样，且无法收发消息。
